scripts/New-VersionConstantsFile.ps1

```diff
@@ -24,17 +25,17 @@ namespace $namespace
 {
     public static class VersionConstants
     {
-        public const string PreRelease = "$versionSuffix";
+        public const string Prerelease = "$versionSuffix";
         public const string AssemblyVersion = "$versionPrefix";
         public const string FileVersion = AssemblyVersion + ".0";
-        public const string Version = AssemblyVersion + PreRelease;
+        public const string Version = AssemblyVersion + Prerelease;
     }
 }
 "@
 
-if (-not (Test-Path $outputDirectory)) {
-    New-Item -ItemType Directory -Path $outputDirectory | Out-Null
-}
```

Comment:
We're generating it in the source directory, so no need to check and create the directory first.